Login: Update styles and layout of QR code screen

ryelle opened the pull request:
Related to #100443, DOTCOM-36

## Proposed Changes

* Remove the second heading
* Remove card styling and align all text next to the QR code
* Add the (w) logo to the top left
* Update the notice styling and text

## Why are these changes being made?

Consistency

## Screenshots

| | Before | After |
|---|---|---|
| Default | ![before-default](https://github.com/user-attachments/assets/e81eaecc-6bfb-476f-ae42-b4116b379853) | ![after-default](https://github.com/user-attachments/assets/358367c2-2886-49ba-8cee-8e15bace3d95)
| Error state | ![before-error](https://github.com/user-attachments/assets/38e53a81-7626-40cf-88b2-1df98f7e7b50) | ![after-error](https://github.com/user-attachments/assets/f781c204-a59b-46d3-9593-05c07e4d2b1f)
| Woo login | ![before-woo](https://github.com/user-attachments/assets/6926bba2-54d7-449f-acc4-9fb4ef4312bf) | ![after-woo](https://github.com/user-attachments/assets/a744534c-797f-4e9e-8970-eab01bd76f23)

Note that the (w) branding appears for the Woo, Akismet, etc login flows as well, rather than the specific branding. IMO, changing that would probably require duplicating code or maybe refactoring how the login screens work (I think), so I didn't pursue that here.

## Testing Instructions

* Go to log in
* Click "Log in via Jetpack app"
* See the QR code

QR logins available for:

- [WordPress.com](http://calypso.localhost:3000/log-in)
- [Woo](http://calypso.localhost:3000/log-in?redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%2F%3Fresponse_type%3Dcode%26client_id%3D50916%26state%3Db30d4e6a81f72b803c873f517f4189601c46a1bc8b9bfdb9ed27cf3d17ad5601%26redirect_uri%3Dhttps%253A%252F%252Fwoocommerce.com%252Fwc-api%252Fwpcom-signin%253Fnext%253D%25252F%26blog_id%3D0%26wpcom_connect%3D1%26wccom-from%26calypso_env%3Dproduction%26from-calypso%3D1&client_id=50916&signup_url=%2Fstart%2Fwpcc%2Foauth2-user%3Foauth2_client_id%3D50916%26oauth2_redirect%3Dhttps%253A%252F%252Fpublic-api.wordpress.com%252Foauth2%252Fauthorize%252F%253Fresponse_type%253Dcode%2526client_id%253D50916%2526state%253Db30d4e6a81f72b803c873f517f4189601c46a1bc8b9bfdb9ed27cf3d17ad5601%2526redirect_uri%253Dhttps%25253A%25252F%25252Fwoocommerce.com%25252Fwc-api%25252Fwpcom-signin%25253Fnext%25253D%2525252F%2526blog_id%253D0%2526wpcom_connect%253D1%2526wccom-from%2526calypso_env%253Dproduction%2526from-calypso%253D1)
- [Akismet](http://calypso.localhost:3000/log-in?redirect_to=https%3A%2F%2Fr-login.wordpress.com%2Fremote-login.php%3Faction%3Dlink%26back%3Dhttps%253A%252F%252Fakismet.com%252Faccount%252F&signup_url=%2Fstart%2Faccount%2Fuser-social%3Fredirect_to%3Dhttps%253A%252F%252Fr-login.wordpress.com%252Fremote-login.php%253Faction%253Dlink%2526back%253Dhttps%25253A%25252F%25252Fakismet.com%25252Faccount%25252F)
- [P2](http://calypso.localhost:3000/log-in?from=p2)

## Pre-merge Checklist

<!--
Complete applicable items on this checklist **before** merging into trunk. Inapplicable items can be left unchecked.

Both the PR author and reviewer are responsible for ensuring the checklist is completed.
-->

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
    - [ ] For UI changes, have we tested the change in various languages (for example, ES, PT, FR, or DE)? The length of text and words vary significantly between languages. 
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

ryelle committed:
Login: Update styles and layout of QR code screen

ryelle committed:
Add the WordPress logo to QR login screen

fditrapani commented:
Thanks for getting this started! Looks so much better already! I have some minor comments:

- When you go from the login page to the this one, there's a big layout shift. The email me a link option is centred and the shift feels less noticeable. Can we vertically centre this layout too?
- Can we update "Tap the Me Tab." to "Tap the **Me** tab." It matches the next point by bolding the name of the thing you have to action on and changes tab to lowercase. 
- Can we left align the title for Woo. I know their screens have the centre aligned title but it looks weird in the column like that. It should look fine with the text centre aligned since the block is centred

ryelle committed:
Update instructions text

ryelle committed:
Fix woo text alignment and link size

ryelle committed:
Center the content vertically on page

ryelle commented:
Here are those updates- centered layout, updated "Me" text, and fixed the Woo layout. I also noticed the link in the instructions was a different font size, so I've fixed that too.

![Screen Shot 2025-03-28 at 12 20 37](https://github.com/user-attachments/assets/7b44453f-1d3d-4b32-8690-71c04150d6d6)

![Screen Shot 2025-03-28 at 12 30 43](https://github.com/user-attachments/assets/2efc7530-cfbb-49cb-ae10-6505b20201f1)

fditrapani reviewed (approved):
Looks great on my end! Thanks for the update

ramonjd commented:
I'll give this a quick test and merge next week. Thanks folks.

a8ci18n commented:
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/17366625

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Thank you @ryelle for including a screenshot in the description! This is really helpful for our translators.
